app/controllers/api/users_controller.rb

```diff
@@ -59,6 +72,19 @@ def destroy
 
     private
 
+    def update_params
+      name = params.fetch(:name, user.name)
+      email = params.fetch(:email, user.email)
+      admin = params.fetch(:admin, user.admin)
+      activated = params.fetch(:activated, user.activated)
+      [name, email, admin, activated]
+    end
+
+    def render_user_existing
```

Comment:
ここ、 @KT-tim に意見を聞きたいなあ。進行形にする必要はないとおもったんだけどどうだろう。
```suggestion
    def render_user_exists
```

Reply:
気になって調べてみたら、進行形は２通りの解釈ができるので最初から避けておこうという主張を見つけました　なんとなく自分の感覚に近いものはあります　進行形にするのは危ないかもしれませんね
https://qiita.com/honey32/items/4ea8833cc8bf5047df6e#2-2-%E6%B3%A8%E5%8B%95%E5%90%8D%E8%A9%9E--ing-%E3%81%AF%E9%81%BF%E3%81%91%E3%82%8B

Reply:
話変わりますが、自分だったらメソッドに切り出しません
重複部分は必ずDRYというのがRailsチュートリアルの教えでしたが、そこまでしなくてもいいのではないかというのが自分の考えです（これについてはもう少し他の人の意見を聞くのが良いと思います）

- 中身２行なので、行数削減の効果が薄い
- メソッドの中身が気になってスクロールとか定義ジャンプする手間や、そのまま迷子になるリスクが生じる